### ISSUE TITLE
Add support for `dreamqltecan`

### ISSUE THREAD

**Issue by BruceFraser1** [openandroidinstaller-dev/openandroidinstaller#610]
### Is your device already requested?

- [X] I have searched the existing issues

### Brand

Samsung

### Name

Galaxy S8

### Device code

dreamltecan

### Model

SM-G950W

### Device images

```Markdown
https://xdaforums.com/t/crdroid-10-2-android-14-for-s8-dreamlte-only-unofficial.4659755
```

### Do you own the device and would be willing to test the installer?

- [X] I own the device and am willing to test the installer on it.

### Additional context

```Markdown
This device appears to be a variant of the Galaxy S8, intended specifically for use in Canada.
OpenAndroidInstaller gives the message that it's not yet supported, and the message says it's "dreamltecan", instead of the usual "dreamlte".
Thanks for your help!
```

**Comment by tsterbak** [comment]
Hm, that should be easy to fix. We will add support for this in the next release :)

**Comment by BruceFraser1** [comment]
Hi Tobias, 
I apologize. I no longer have that model phone, but did not remove my request. Sorry to be a bother to you. 

⁣Regards,
Bruce ​

**Comment by SirRGB** [comment]
Well Im also not quite convinced that the rom supports your model as per https://github.com/8890q/android_device_samsung_dreamlte/blob/f885b96f68f7c9579134a316206e3a556a3f870a/BoardConfig.mk#L22

**Comment by SirRGB** [comment]
https://github.com/openandroidinstaller-dev/openandroidinstaller/issues/505

Is dreamq* the qcom variant @tsterbak ?
The title says dreamqltecan, everything else dreanltecan. Kinda doubt they sell 2 diff soc variants of the same phone in one region.

**Comment by tsterbak** [comment]
Yeah, you are right... Looks like its unclear if dreamlte ROMs would work on dreamltecan devices... Maybe I will remove it again from the PR.
dreamqltecan would indicate a snapdragon soc, while dreamltecan would most likely be an Exynos soc...
